TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Components/IntakeLift.java

```diff
@@ -0,0 +1,67 @@
+package org.firstinspires.ftc.teamcode.Components;
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.Servo;
+
+/**
+ * Intake Lift Controls:
+ * Gamepad: 2
+ * Joystick: Both
+ */
+public class IntakeLift {
+
+    OpMode opMode;
+
+    DcMotor primaryMotor;
+    Servo secondaryMotor;
+
+    public enum AutoPos{
+        UP(0,0), DOWN(0,0); //TODO: TEST FOR VALUES
+
+        private final int primaryMotorPos;
+        private final int secondaryMotorPos;
+
+        AutoPos(int primaryMotorPos, int secondaryMotorPos) {
+            this.primaryMotorPos = primaryMotorPos;
+            this.secondaryMotorPos = secondaryMotorPos;
+        }
+
+        public int getPrimaryMotorPos() {
+            return primaryMotorPos;
+        }
+
+        public int getSecondaryMotorPos() {
+            return secondaryMotorPos;
+        }
+    }
+
+    public IntakeLift(OpMode opMode){
+        this.opMode = opMode;
+    }
+
+    public void init(boolean debug){
+        primaryMotor = opMode.hardwareMap.dcMotor.get("PrimaryMotor");
+        secondaryMotor = opMode.hardwareMap.servo.get("SecondaryMotor");
+
+        if(debug){
+            opMode.telemetry.addData("INFO","Intake Lift INIT");
+        }
+    }
+
+    public void run(){
+        primaryMotor.setPower(opMode.gamepad2.left_stick_y);
+        setSpeed(secondaryMotor, opMode.gamepad2.right_stick_y);
+    }
+
+    public void setAutoPos(AutoPos autoPos){
+        primaryMotor.setTargetPosition(autoPos.getPrimaryMotorPos());
+        secondaryMotor.setPosition(autoPos.getSecondaryMotorPos());
```

Comment:
Servos are either set to continuous, or position mode. This is done by the servo configurator and not in code, so I don't think you can have it do both in software, unfortunately.

Reply:
@GarrettBurroughs Are we unable to set position in continuous? If so what is the downside from a position mode servo for the secondary motor for the intake lift? I do not see any from my persepective.

Reply:
in the teleop code you set the power (indicating using contiunous mode), whereas in auto you set the position (indicating using posision mode). It is doable using only a position servo, but you would have to change the code, I can implement that now if you want

Reply:
@GarrettBurroughs Double check the secondary motor. It is using set speed which is a method that runs off of position. The primary motor is the dc motor. Regarding teleop btw.

Reply:
@H1ppx 
```java
    private void setSpeed(Servo servo, double speed){
        servo.setPosition(speed/2+0.5);
    }
```
https://github.com/MontclairRobotics/RoverRuckus/blob/f1f58c4291b9f2b2e7c89d51d7095240982408ff/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Components/IntakeLift.java#L57

This only works when in continuous mode

Reply:
@GarrettBurroughs Are you sure, It should work within the bounds of a non continuous servo with it stopping at the bounds